### PR TITLE
Add Randomplayer class

### PR DESCRIPTION
**Pull request by WanleiZhang**
I have written the random player as a class and updated the main.py to add a random player in the game.

### CODE REVIEW

File: players/Randomplayer.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+from .Player import Player
+import random
+
+
+class Randomplayer(Player):
+
+    """A class representing a random player.
+        A random player who always guess randomly with the flipped coin game.
```
Comment (MichaelCouch) [inline players/Randomplayer.py]: * Should work for any game, not just CoinFlip
* Let's keep this super tidy. Please add a line break to it's <80 characters wide
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

File: players/Randomplayer.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        for player_id, score in observation['scores'].items():
+            print(f"{player_id}: {score}")
+
+        print("The game looks like")
+        print(observation['observation'])
+
+        return self.read_move(observation['moves'])
+
+    def read_move(self, moves):
+        """Get the next player move, with checking
```
Comment (MichaelCouch) [inline players/Randomplayer.py:38]: Can we merge this into the select_move method? I don't think we need an extra helper function here.

File: players/Randomplayer.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+        :moves: list of strings
+        :returns: element in moves
+
+        """
+        
+        move = random.choice(moves)
```
Comment (MichaelCouch) [inline players/Randomplayer.py]: Can we delete read_move and merge this into the select_move method? I don't think we need an extra helper function here. read_move in Human was to separate out the logic of making sure the user submits a valid move, but hopefully the random player never makes an invalid move

**Review by MichaelCouch — CHANGES_REQUESTED** [review]
Code looks good, but I left a couple of comments. Being extra fussy as we get started.

File: players/Randomplayer.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+
+from .Player import Player
+import random
+
+
+class Randomplayer(Player):
```
Comment (MichaelCouch) [inline players/Randomplayer.py]: Usually we use TitleCase for python classes, e.g. DataFrame, CoinFlip

File: players/Randomplayer.py (hunk 1 above)
Comment (WanleiZhang) [reply players/Randomplayer.py]: Do you use any particular code formatter to do this or you just manually format it to make sure it doesn't go over the minimum line length?

Comment (MichaelCouch) [reply players/Randomplayer.py]: I use pyflakes embedded into my text editor. Maybe I also use flake8. It checks the python you write conforms to pep8 standard

File: players/Randomplayer.py (hunk 4 above)
Comment (WanleiZhang) [reply players/Randomplayer.py]: Noted and updated!

File: players/Randomplayer.py (hunk 1 above)
Comment (WanleiZhang) [reply players/Randomplayer.py]: Updated!

File: players/Randomplayer.py (hunk 3 above)
Comment (WanleiZhang) [reply players/Randomplayer.py]: Yes, I have removed read_move and merged into select_move for the randomplayer class.

**Review by MichaelCouch — APPROVED** [review]

**Pull request merged**